src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -541,6 +543,44 @@ private static bool ProcessAnyFileContent(IRequestInfo info, string localPath)
 			return true;
 		}
 
+		private bool ProcessCssFile(IRequestInfo info, string localPath)
+		{
+			if  (CurrentBook == null) return false;
```

Comment:
How will this affect working on books that aren't the current book, such as when making thumbnails? Bloom starts doing that before a current book is even selected.